pkg/agent/plugin/nodeattestor/aws/iid.go

```diff
@@ -19,8 +21,11 @@ import (
 )
 
 const (
-	docPath = "instance-identity/document"
-	sigPath = "instance-identity/signature"
+	docPath                 = "instance-identity/document"
+	sigPath                 = "instance-identity/signature"
+	loadConfigTimeout       = 30 * time.Second
+	fetchMetadataDocTimeout = 30 * time.Second
+	fetchMetadataSigTimeout = 30 * time.Second
```

Comment:
Any reason we want individual timeouts for these operations? In general, I think we should probably leave this up to SPIRE core by threading through the context passed into the RPC handler.

Reply:
Not really - agreed that we should be using the stream context here